appknox/cli.py

```diff
@@ -17,6 +21,7 @@
 
 CONFIG_FILE = os.path.expanduser('~/.config/appknox.ini')
 DEFAULT_PROFILE = 'default'
+TOKEN_EXPIRY_DAYS = 7
```

Comment:
shit you are right @dhilipsiva 

although we are changing this with oauth application for `cli` and `devknox` so that we can enable MFA and using the oauth token, which expires according to the oauth rules